csm/conf/setup.py

```diff
@@ -71,14 +72,16 @@ def _run_cmd(cmd):
         Run command and throw error if cmd failed
         """
         try:
+            Log.info(f"Executing command: {cmd}")
             _err = ""
             _proc = SimpleProcess(cmd)
             _output, _err, _rc = _proc.run(universal_newlines=True)
             if _rc != 0:
                 raise
             return _output, _err, _rc
         except Exception as e:
-            raise CsmSetupError("Csm setup is failed Error: %s %s" %(e,_err))
+            Log.error(f"Csm setup is failed Error while executing {cmd}: {e} {_err}")
```

Comment:
Csm setup is failed Error while executing -> CSM setup failed. Error while executing

Reply:
Comment Address